Cargo.toml

```diff
@@ -85,6 +85,7 @@ clap = "4.2.2"
 codegen = "0.2.0"
 criterion = "0.4.0"
 ctrlc = "3.1"
+derive_more = "=1.0.0-beta.6"
```

Comment:
Unfortunately, the version of `derive_more` already used in the workspace (0.99.17) doesn't have good `Debug` derivation support.